pychord/constants/qualities.py

```diff
@@ -25,6 +26,7 @@
     ('7sus4', (0, 5, 7, 10)),
     ('m6', (0, 3, 7, 9)),
     ('m7', (0, 3, 7, 10)),
+    ('m7b5', (0, 3, 5, 10)),
```

Comment:
I think `m7b5` or `m7(b5)` is as same as `m7-5`, isn't it?

```suggestion
    ('m7-5', (0, 3, 6, 10)),
```

Reply:
Ah. I wasn't sure whether those symbols are the same. Looks like there are some usages for `b5` & `-5` in some places like here: https://github.com/dok/pychord/blob/663b84e981a7113f07865aefc8a3c98f1d760dee/pychord/constants/qualities.py#L22-L23